src/commands/app/undeploy.js

```diff
@@ -46,12 +47,16 @@ class Undeploy extends BaseCommand {
     const spinner = ora()
     try {
       const aioConfig = (await this.getFullConfig()).aio
-      const cliDetails = await getCliInfo()
+      const cliDetails = doesTokenExists ? await getCliInfo() : null
       const logEvent = getAuditLogEvent(flags, aioConfig.project, 'AB_APP_UNDEPLOY')
 
       // 1.1. send audit log event for successful undeploy
-      if (logEvent) {
-        await sendAuditLogs(cliDetails.accessToken, logEvent, cliDetails.env)
+      if (logEvent && cliDetails) {
+        try {
+          await sendAuditLogs(cliDetails.accessToken, logEvent, cliDetails.env)
+        } catch (error) {
+          this.log(chalk.red(chalk.bold('Error: Audit Log Service Error: Failed to send audit log event for deployment.')))
```

Comment:
it is an error in sending, but to the cli is it an error? I would say it's more like a warning here. Just do a `this.warn` without all the `chalk` adornment

Typo: should be `undeployment` (is that even a word?)